src/lib/rest/rest.cpp

```diff
@@ -789,7 +789,6 @@ static int connectionTreat
     if (!ciP->httpHeaders.servicePathReceived)
     {
       ciP->httpHeaders.servicePath = defaultServicePath(url, method);
-      ciP->httpHeaders.servicePathReceived = true;
```

Comment:
This here was a mistake from not so long ago.
I saw it in the PR but I let it pass, didn't think it mattered.
However, for this fix I need to know whether the servicePath is actually path of the request or not, so I had to remove it.

Just informative.
NTC